src/bin/server.rs

```diff
@@ -21,6 +21,19 @@ fn main() -> Result {
         return static_web_server::settings::cli_output::display_version();
     }
 
+    if let Ok(path) = std::env::var("SWS_GENERATE") {
```

Comment:
I would like to see this functionality behind a new subcommand so it does not get mixed with the server options.

An idea could be:

- `static-web-server generate --completions <OUT_DIR>`
- `static-web-server generate --man-page <OUT_DIR>`
- `static-web-server generate  <OUT_DIR>` (generate both)

What do you think?

Reply:
Sounds good to me!